dependencies/wasm/CMakeLists.txt

```
@@ -16,7 +16,7 @@ if ("${CMAKE_HOST_SYSTEM_NAME}" STREQUAL "Windows")
 endif ()
 
 if (WITH_WABT)
-    set(WABT_VER 1.0.25)
+    set(WABT_VER main)
```

I am worried about this making our builds unpredictable. I think better would be to hard-code an override-able git commit hash, like so:
```suggestion
    set(WABT_VER "09c40635207d42dd30ffaca22477fd3491dd9e7d"
        CACHE STRING "WABT commit to download (tag, branch name, or hash)")
```
and then in CI we can pull nightlies by setting the `WABT_VER` cache variable to `main`.

---

(`09c40635207d42dd30ffaca22477fd3491dd9e7d` is the current top-of-tree hash)

---

Fair, keep in mind this should be very short term